docs/detections/api/rules/signals-api-overview.asciidoc

```diff
@@ -353,4 +353,68 @@ Example response:
         }
     ]
 }
---------------------------------------------------
\ No newline at end of file
+--------------------------------------------------
+
+==== Assign or unassign users from alerts
+
+Allows you to assign and unassign users from alerts. 
+
+===== Request URL
+
+`POST <kibana host>:<port>/api/detection_engine/signals/assignees`
+
```

Comment:
Can we add the request body schema?

Reply:
Resolved in [205d2ed](https://github.com/elastic/security-docs/pull/4475/commits/205d2ed9d98447f666cf0d1ae3a5f45da6d90c78).